game/views.py

```
@@ -672,22 +737,74 @@ def register_view(request):
+                not settings.EMAIL_HOST_USER or
+                not settings.EMAIL_HOST_PASSWORD
+            )
+
+            # Send relevant alerts/notifications via email
+            if not (settings.DEBUG and missing_email_credentials):
+                try:
+                    if conflict_type in ('exact_active', 'email_active'):
+                        email_to = email
+                        subject = 'Checkora Registration Attempt'
+                        msg_plain = (
+                            f"Hello! Someone tried to register a Checkora account using your email address.\n\n"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)